spec/rubocop/cop/style/frozen_string_literal_comment_spec.rb

```diff
@@ -164,120 +249,21 @@
-
-        expect(new_source).to eq(<<~RUBY)
-          # frozen_string_literal: true
-          puts 1
-        RUBY
-      end
-
-      it 'adds a frozen string literal comment to the first line if one is ' \
-         'missing and handles extra spacing' do
-        new_source = autocorrect_source(<<~RUBY)
-
-          puts 1
-        RUBY
-
-        expect(new_source).to eq(<<~RUBY)
-          # frozen_string_literal: true
-
-          puts 1
-        RUBY
-      end
-
-      it 'adds a frozen string literal comment after a shebang' do
-        new_source = autocorrect_source(<<~RUBY)
-          #!/usr/bin/env ruby
-          puts 1
-        RUBY
-
-        expect(new_source).to eq(<<~RUBY)
-          #!/usr/bin/env ruby
-          # frozen_string_literal: true
-          puts 1
-        RUBY
-      end
-
-      it 'adds a frozen string literal comment after an encoding comment' do
-        new_source = autocorrect_source(<<~RUBY)
-          # encoding: utf-8
-          puts 1
-        RUBY
-
-        expect(new_source).to eq(<<~RUBY)
-          # encoding: utf-8
-          # frozen_string_literal: true
-          puts 1
-        RUBY
-      end
-
-      it 'adds a frozen string literal comment after a shebang and encoding ' \
-         'comment' do
-        new_source = autocorrect_source(<<~RUBY)
-          #!/usr/bin/env ruby
-          # encoding: utf-8
-          puts 1
-        RUBY
-
-        expect(new_source).to eq(<<~RUBY)
-          #!/usr/bin/env ruby
-          # encoding: utf-8
-          # frozen_string_literal: true
-          puts 1
-        RUBY
-      end
-
-      it 'adds a frozen string literal comment after a shebang and encoding ' \
-         'comment when there is an empty line before the code' do
-        new_source = autocorrect_source(<<~RUBY)
-          #!/usr/bin/env ruby
-          # encoding: utf-8
-
-          puts 1
-        RUBY
-
-        expect(new_source).to eq(<<~RUBY)
-          #!/usr/bin/env ruby
-          # encoding: utf-8
-          # frozen_string_literal: true
-
-          puts 1
-        RUBY
-      end
-
-      it 'adds a frozen string literal comment after an encoding comment ' \
-         'when there is an empty line before the code' do
-        new_source = autocorrect_source(<<~RUBY)
-          # encoding: utf-8
-
-          puts 1
-        RUBY
-
-        expect(new_source).to eq(<<~RUBY)
-          # encoding: utf-8
-          # frozen_string_literal: true
-
-          puts 1
-        RUBY
-      end
-
-      it 'adds a frozen string literal comment after a shebang when there is ' \
-         'only a shebang' do
-        new_source = autocorrect_source(<<~RUBY)
-          #!/usr/bin/env ruby
-        RUBY
-
-        expect(new_source).to eq(<<~RUBY)
-          #!/usr/bin/env ruby
-          # frozen_string_literal: true
-        RUBY
-      end
+    it 'registers an offence for an extra first empty line' do
+      pending 'There is a flaw that skips adding caret symbol in this case' \
```

Comment:
Couldn't find a quick way around it. What's even stranger, can't find a difference in `loc` being added in case when caret is added or not.